docs/developer_docs.rst

```diff
@@ -30,24 +48,34 @@ Classes
 .. autoclass:: pyuvdata.fhd.FHD
   :members:
 
+.. autoclass:: pyuvdata.ms.MS
+  :members:
+
+.. autoclass:: pyuvdata.uvh5.UVH5
+  :members:
+
+.. autoclass:: pyuvdata.fhd_cal.FHDCal
+  :members:
+
 .. autoclass:: pyuvdata.calfits.CALFITS
   :members:
 
 .. autoclass:: pyuvdata.beamfits.BeamFITS
   :members:
 
-.. autoclass:: pyuvdata.Telescope
+.. autoclass:: pyuvdata.cst_beam.CSTBeam
   :members:
 
 
 Functions
 ----------
-.. autofunction:: pyuvdata.known_telescopes
 
-.. autofunction:: pyuvdata.get_telescope
+.. autofunction:: pyuvdata.version.construct_version_info
+
+.. autofunction:: pyuvdata.fhd.get_fhd_history
 
+aipy extracts
+_____________
 
-Utility Functions
------------------
-.. automodule:: utils
+.. automodule:: pyuvdata.aipy_extracts
```

Comment:
I don't think these are rendering correctly on RTD:
![image](https://user-images.githubusercontent.com/4752917/58209936-eaf24880-7c9c-11e9-8ad7-df5688104a83.png)

Reply:
oops, good catch! It should be fixed now (as soon as RTD updates)